karma.conf.js

```diff
@@ -0,0 +1,114 @@
+process.env.SAUCE_USERNAME = 'shard'
+var im_lazy_dot_tiff = '372040cd-d46b-4a2d-9d8b-cab91ef9efaa'
+process.env.SAUCE_ACCESS_KEY =  im_lazy_dot_tiff //shh, don't tell anyone
```

Comment:
I have created a Sauce Labs account for this project, and have added `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY` to the CircleCI configuration. You could remove this now and see whether the new account is configured correctly.

Also, you should generate a new access key for your Sauce Labs account. ;)